plugins/serverguardsupport.lua

```diff
@@ -0,0 +1,11 @@
+PLUGIN.name = "ServerGuard Support"
+PLUGIN.author = "Sample Name"
+PLUGIN.desc = "Disables ServerGuard's restrictions plugin"
+PLUGIN.players = {}
```

Comment:
What it is? Why is it here?

Reply:
Thanks for pointing it out. I took the template from another plugin and didn't notice it was there
Pushed a new commit